.vsts-pipelines/builds/ci-official.yml

```diff
@@ -22,6 +31,15 @@ jobs:
       TeamName: AspNetCore
       OfficialBuild: true
       VisualStudioMSBuildx86Path: 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\MSBuild.exe'
+      # CodeQL3000 tasks are auto-injected into official builds of main. Variables below are ignored otherwise.
+      # Enable CodeQL3000 auto-injected tasks. These tasks will no-op if two builds run w/in the same week.
+      Codeql.Cadence: 168
```

Comment:
Why 168?

Reply:
One week. I mentioned that value later in the "Action Required: Enable CodeQL3000 in your pipelines ..." thread for builds that don't run frequently. Should make the database seem more "current" more of the time.

Reply:
Could you add a comment here mentioning that it'll run weekly?

Reply:
Isn't "These tasks will no-op if two builds run w/in the same week." sufficient❔